tensorflow/python/kernel_tests/linalg/sparse/csr_sparse_matrix_dense_mat_mul_onednn_grad_test.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Do you actually need a separate file/test here?  Can't you just create a new py test and set the `env` property?

Reply:
I remember that we saw `tensorflow/python/kernel_tests/nn_ops/cudnn_deterministic_ops_test.py` (or similar, it's been a while) and decided to adopt its strategy for setting an environment variable.  I'm testing the set-environment-in-Bazel strategy now; it doesn't make a difference to me which one we use.

Reply:
I'd prefer the simplicity of a single test file, with different settings for `env` in the test definition.

Reply:
All right, good idea. I remember that approach failing locally for some reason -- I don't recall what the issue was -- but it works now, so just pushed the change back to a single test file.